perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -56,6 +56,26 @@ use constant WHITESPACE_UNDEF_DATE => '0001-01-01T00:00:00';
 use constant MISSED_COLLECTION_SERVICE_PROPERTY_ID => 68;
 use constant REQUEST_SERVICE_PROPERTY_ID => 69;
 
+sub fetch_whitespace_data {
+    my ($self, $method, $uprn) = @_;
+
+    my $c = $self->{c};
+
+    my $async = $c->action eq 'waste/bin_days'
+        && $c->req->method eq 'GET';
+
+    my $data = $self->whitespace->call_api(
```

Comment:
Note that this call stores its results in the user's session, so a corresponding clear_cached_lookups_property call needs to be added so that it's cleared in appropriate places (e.g. when the bin day page is visited anew).